src/bugsnag/client.ts

```diff
@@ -1527,6 +1527,145 @@ export class BugsnagClient implements Client {
         };
       },
     );
+
+    const getNetworkGroupingInputSchema = z.object({
+      projectId: toolInputParameters.projectId,
+    });
+
+    register(
+      {
+        title: "Get Network Endpoint Groupings",
+        summary: "Get the network endpoint grouping rules for a project",
+        purpose:
+          "Retrieve the URL patterns used to group network spans for performance monitoring",
+        useCases: [
+          "View current network endpoint grouping configuration",
+          "Understand how network requests are being grouped in performance monitoring",
+          "Check grouping patterns before making updates",
+        ],
+        inputSchema: getNetworkGroupingInputSchema,
+        examples: [
+          {
+            description: "Get network grouping rules for a project",
+            parameters: {},
+            expectedOutput:
+              "JSON object with projectId and array of endpoint URL patterns",
```

Comment:
Shall we just return the array of URLs? The containing object seems a bit unnecessary